### DISCUSSION TITLE
Link to canceling workflow

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12332] [Actions] [answered]
https://github.com/check-spelling/presto/actions/runs/1926914391

> [test-other-modules](https://github.com/check-spelling/presto/runs/5404060820?check_suite_focus=true)
Canceling since a higher priority waiting request for 'test other modules-1' exists

It'd be really nice if this linked to the higher priority waiting request.

> [test-other-modules](https://github.com/check-spelling/presto/runs/5404060820?check_suite_focus=true)
The operation was canceled.

Also note that this vaguely related thing ends with a `.`. It'd be nice if related things used punctuation consistently.

**Comment by ethomson** [comment, marked as answer]
Thanks @jsoref - seems like a reasonable request, I'll pass this along to the team.
